constraints.txt

```diff
@@ -0,0 +1 @@
+torch==2.6.0
```

Comment:
Suggestion: Since this file is only used for constraining versions in tests, how about you rename the file to `constraints-dev.txt` and explain the purpose in the a comment?

Reply:
Yep, great idea! I'll do that

Reply:
Renamed this file to `constraints-dev.txt`!